app/assets/stylesheets/onboarding.scss

```diff
@@ -0,0 +1,66 @@
+@import 'variables';
+@import 'mixins';
+@import 'config/import';
+
+.onboarding-task-card {
+  background: #3b49df;
+  border-radius: $br-default;
+  padding: $su-7 $su-8 $su-8 $su-8;
+  display: none;
+
+  .task-card-title {
+    font-weight: $fw-heavy;
+    font-size: $fs-4xl;
+    line-height: $lh-tight;
+    color: #ffffff;
+  }
+
+  .task-card-subtitle {
+    color: #ffffff;
+    opacity: 0.7;
+    font-weight: $fw-medium;
+    font-size: $fs-s;
+    line-height: $lh-tight;
+    letter-spacing: -0.02em;
+  }
+
+  .task-card-action {
+    border-radius: $br-default;
+    background-color: rgba(255, 255, 255, 0.2);
+
+    &:hover {
+      background-color: rgba(255, 255, 255, 0.3);
+    }
+  }
+
+  a.task-card-link {
+    display: block;
+    border-radius: $br-default;
+    padding: $su-5 $su-5;
+    margin-top: $su-4;
+    font-weight: $fw-bold;
+    font-size: $fs-base;
+    line-height: $lh-base;
+    letter-spacing: -0.02em;
+    color: #ffffff;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &:hover {
+      cursor: pointer;
+    }
+
+    .emoji {
+      margin-right: $su-2;
+    }
+  }
+
+  .close {
+    background: none;
+    border: none;
+    position: absolute;
```

Comment:
are these styles overriding something else? i can't try this out locally so i don't know for sure...but they seem to be "unsetting" some other style. 

if so, we should make a comment.

Reply:
tried this out locally and it looks like they are necessary to style the button, so you can ignore this comment!

Reply:
Yeah, they are overriding buttons. Added!